pkg/kubectl/describe.go

```diff
@@ -524,6 +525,7 @@ func describeService(service *api.Service, endpoints *api.Endpoints, events *api
 			}
 			fmt.Fprintf(out, "Endpoints:\t%s\t%s\n", name, formatEndpoints(endpoints, util.NewStringSet(sp.Name)))
```

Comment:
I suggest updating formatEndpoints to count endpoints to display a count in the case that it can't display the full list.